memory-game-project/src/styles.scss

```diff
@@ -0,0 +1,16 @@
+$green: #8cff66;
+$font: Lucida Sans Unicode;
+
+body {
+  font-family: $font;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+#appendDiv {
+  background-color: $green;
+  padding: 20px;
+  margin: 20px;
+  border: 2px solid black;
```

Comment:
avoid using id in selectors. It is always better to use class instead. also scss give you an ability to *connect* styles from the same block. e.g. following is equals to your definition:
```
body {
  font-family: $font;
  display: flex;
  justify-content: center;
  align-items: center;
  
  #appendDiv {
    background-color: $green;
    padding: 20px;
  }
}
```

so when you are creating styles it is better to understand the structure, as for me.

also scss is very helpful when using [BEM](http://getbem.com/naming/)

e.g.
```
.lovely-header {
  &__button {
    &--active {
    }
  }
}
```
will generate 3 styles:
```
.lovely-header {
 }

.lovely-header__button {
}

.lovely-header__button--active {
}
```